### PR TITLE
add independent nq limit option

### PR DESCRIPTION
**Pull request by chyezh**
Related Issue: #24611

### CODE REVIEW

**Comment by zhuwenxing** [comment]
Why do you skip e2e when your PR includes source code modification?

File: internal/util/paramtable/component_param.go [hunk 1]
```diff
@@ -460,6 +462,10 @@ func (p *commonConfig) initTopKLimit() {
 	p.TopKLimit = p.Base.ParseInt64WithDefault("common.topKLimit", 16384)
 }
 
+func (p *commonConfig) initNQLimit() {
+	p.NQLimit = p.Base.ParseInt64WithDefault("common.nqLimit", 16384)
```
Comment (bigsheeper) [inline internal/util/paramtable/component_param.go:466]: move to `quotaAndLimits.limits` ?

**Comment by bigsheeper** [comment]
/lgtm

lgtm if common.TopKLimit is not set in cloud.

**Comment by chyezh** [comment]
> /lgtm
> 
> lgtm if common.TopKLimit is not set in cloud.

already confirmed with @yelusion2

**Pull request merged**